Dockerfile

```diff
@@ -25,7 +25,7 @@ FROM base as build
 RUN apt-get install --no-install-recommends -y build-essential git libpq-dev pkg-config
 
 # Install application gems
-COPY Gemfile Gemfile.lock ./
+COPY Gemfile Gemfile.lock ./ .ruby-version
```

Comment:
```suggestion
COPY .ruby-version Gemfile Gemfile.lock ./
```

Reply:
The order of the `COPY` command matters here—all the source files must come before the destination, `./`